torchreid/models/efficient_net_pytcv.py

```diff
@@ -399,7 +400,7 @@ def forward(self, x, return_featuremaps=False, get_embeddings=False):
         else:
             raise KeyError("Unsupported loss: {}".format(self.loss))
 
-        if self.lr_finder.enable and self.lr_finder.mode == 'automatic':
+        if self.lr_finder.enable and self.lr_finder.mode == 'fast_ai':
```

Comment:
These ifs look strange: you have to hack several models to fit one finder. I propose to hack one finder to fit all the models and get rid of `self.lr_finder.enable`-related conditions here. You could use an additional nn.Module wrapper that converts a tuple to a tensor.